auctions/english-auction/project/contracts/auction-contract/tests/utils/setup.rs

```diff
@@ -14,19 +14,19 @@ abigen!(
     ),
     Contract(
         name = "Nft",
-        abi = "./contracts/auction-contract/tests/artifacts/NFT/out/debug/NFT-abi.json"
+        abi = "./contracts/test-artifacts/NFT/out/debug/NFT-abi.json"
```

Comment:
Similar to other PRs, the asset should be moved back or re-written to use the SDK while the NFT should be inside the tests dir rather than moved up.

Reply:
This is required due to changes with nested projects and forc workspaces. An error is produced if the test artifacts are kept in their old places.

Reply:
Have written to use SDK